dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/utils/DockerfileGitHubUtil.java

```diff
@@ -544,4 +569,20 @@ public void changeDockerfiles(Namespace ns,
                     rateLimiter);
         }
     }
+
+    public static boolean isValidImageTag(String tag) {
+        String tagVersionRegexStr = "([a-zA-Z0-9_]([-._a-zA-Z0-9])*)";
+        Pattern validTag = Pattern.compile(tagVersionRegexStr);
+        if (StringUtils.isNotBlank(tag)) {
+            if (!validTag.matcher(tag.trim()).matches()) {
+                log.warn("{} is not a valid value for image tag. So will be ignored!", tag);
+                return false;
+            } else if (tag.trim().equals("latest")) {
```

Comment:
I think we'll want to parameterize this at some point. For now this is good to have in this PR